client/homebrew/editor/editor.jsx

```diff
@@ -82,11 +83,49 @@ const Editor = createClass({
 		if(!this.refs.codeEditor) return;
 		const codeMirror = this.refs.codeEditor.codeMirror;
 
+		//Clear all Text styles
+		const ranges=codeMirror.getAllMarks();
+		for (let i=0;i<ranges.length;i++) ranges[i].clear();
+
 		const lineNumbers = _.reduce(this.props.value.split('\n'), (r, line, lineNumber)=>{
+
+			//clear all line styles
+			codeMirror.removeLineClass(lineNumber, 'background');
+			codeMirror.removeLineClass(lineNumber, 'text');
+
 			if(line.indexOf('\\page') !== -1){
 				codeMirror.addLineClass(lineNumber, 'background', 'pageLine');
 				r.push(lineNumber);
 			}
+
+			// New Codemirror styling for V3 renderer
+			if(this.props.renderer == 'V3') {
+				if(line.indexOf('\\column') === 0){
```

Comment:
I would assume that [`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) should be faster and probably easier to read in the code

Reply:
Yes I agree. I will change this.

Reply:
Done.